x-pack/plugins/security/server/lib/validate_config.js

```diff
@@ -6,10 +6,6 @@
 
 const crypto = require('crypto');
 
-const isDefault = (config, key) => {
-  return config.getDefault(key) === config.get(key);
```

Comment:
Can we remove [config.getDefault](https://github.com/kobelb/kibana/blob/index-application/src/server/config/config.js#L146) altogether, since we aren't relying on it here anymore? IIRC, we introduced that for RBAC Phase 1, and doing a quick scan, I couldn't find any other references to it.

Reply:
You're right, and we could potentially remove it now that we aren't using it anymore... I'll go ahead and make this part of this PR as well, I don't have a good reason to keep it around.